Podfile.lock

```diff
@@ -1,15 +1,20 @@
 PODS:
-  - Nimble (7.0.3)
-  - Quick (1.2.0)
+  - Nimble (7.3.2)
+  - Quick (1.3.2)
 
 DEPENDENCIES:
   - Nimble
   - Quick
 
+SPEC REPOS:
+  https://github.com/cocoapods/specs.git:
+    - Nimble
+    - Quick
+
 SPEC CHECKSUMS:
-  Nimble: 7f5a9c447a33002645a071bddafbfb24ea70e0ac
-  Quick: 58d203b1c5e27fff7229c4c1ae445ad7069a7a08
+  Nimble: 5e94a8cacb355943c6c87198035dc4adbb0f10e4
+  Quick: 2623cb30d7a7f41ca62f684f679586558f483d46
 
 PODFILE CHECKSUM: 0ff3cbcf4867a5e4becef729c12e0a780b03a1a9
 
-COCOAPODS: 1.4.0
+COCOAPODS: 1.5.3
```

Comment:
Probably don't need a lockfile anymore

Reply:
Doh, removed from this and https://github.com/DuetHealth/DrX/pull/9

Also updated all 3 of this MRs to specify platform iOS for CI builds, which I think is fair